test/test_java_primitives.py

```diff
@@ -8,6 +8,8 @@ def test_basic(self):
     def test_basic_args(self):
         code = 'System.out.println("s");\n 1;'
         self.assertEqual('System.out.println("s")\n1', main.main(code, 'java', 'java'))
-
+    def test_basic_var_declaration(self):
+        code = 'int x = 1;\n String s = "s";\nboolean b = false;'
+        self.assertEqual('int x = 1\nString s = "s"\nboolean b = false', main.main(code, 'java', 'java'))
```

Comment:
I would split this into multiple tests to improve readability and make it easier to fix future errors